code/game/objects/structures/shower.dm

```diff
@@ -65,6 +73,17 @@
 	else
 		return ..()
 
+/obj/machinery/shower/multitool_act(mob/living/user, obj/item/I)
```

Comment:
I could be wrong, but aren't we moving towards using `.tool_behaviour` and `TOOL_MULTITOOL`? (Putting aside how it's weird you're using a multitool on a shower, you'd think it'd be just another option when using a wrench?)

Reply:
I was thinking that we were moving in the other direction honestly. Also, the wrench is currently being used for adjusting the temperature of the shower. I don't want to stick an input or radial menu between people and their intended operation when they try to adjust the shower.